apps/Typescript-ObjC/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Similar question to above? Is this a CIO standard? Default in these files? Crowd-sourced best practice?

Reply:
This is not what is CIO standard. This file was auto-generated by the community template. 

I deleted this file because if we want to create one, we should create one that is the same for all our sample apps.